client/components/sidebar.vue

```diff
@@ -91,6 +139,31 @@
     },
 
     methods: {
+
+      showReplies(comment) {
+        comment.showAllReplies = true; // eslint-disable-line no-param-reassign
+        this.layoutCommentsAfterRender();
+      },
+
+      collapseComments() {
+        this.documentComments = this.documentComments.map((c) => {
+          return Object.assign({}, c, {
+            showAllReplies: false,
+          });
+        });
+        this.layoutCommentsAfterRender();
+      },
+
+      onReply(comment) {
+        Comment.create({
+          highlightKey: comment.highlightKey,
+          body: comment.reply,
+          documentId: this.documentId,
+          replyTo: comment._id,
```

Comment:
So this is ID of the top-level comment, yes? The one without `replyTo`. Not the previous comment in the thread? That is good.

Reply:
yes, it's the top-level comment ID